Tests/sbgTests/FoundationFileAdderTests.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  FoundationFileAdderTests.swift
+//  SBGTests
+//
+//  Created by Dawid Markowski on 15/10/2018.
+//
+
+
+import Foundation
+import Quick
+import Nimble
+@testable import SBGCore
+
+class FoundationFileAdderTests: QuickSpec {
+    
+    override func spec() {
+        describe("FoundationFileAdder") {
+            
+            var sut: FoundationFileAdder!
+            var pathResolver: MockPathResolver!
+            
+            beforeEach {
+                pathResolver = MockPathResolver()
+                sut = FoundationFileAdder(pathResolver: pathResolver)
+            }
+            
+            context("when adding file") {
```

Comment:
`describe("add file")` is more sufficient here (`context` is more suitable when specific case/configuration is verified).